### DISCUSSION TITLE
Change the icon "okay"  to "search" symbol in the  'Show POI overlay' menu (Android)

### DISCUSSION THREAD

**Discussion by anastasiia936** [osmandapp/OsmAnd#12201] [Ideas]
In the Configure map->POI overlay. The symbol for POI-filter on bottom left of the window could be more intuitive:
Instead of "okay"-symbol it would be better to have an ordinary "search"-symbol (magnifying glass).
![20210721_162226](https://user-images.githubusercontent.com/80623554/126622176-d5813f76-b1c6-4a2d-bf5c-f28e35e9c1af.jpg)

**Comment by dmpr0** [comment]
"Okay" icon turns on/off multi selection mode in the dialog. I don't understand why it should be a "search" icon if there are no search options.

**Reply by Lee-Carre** [reply]
Indeed; it's not really searching (in the usual sense of that word, within OsmAnd), it's merely highlighting the selected PoIs.

---

Separately; the arrow pointing to the ☑ in the OP's display-capture image, is for toggling multi-select versus single-select modes.
